app/src/main/java/de/mytfg/app/android/api/MytfgApi.java

```diff
@@ -36,6 +40,7 @@ public static void call(String apiFunction, ApiParams params, ApiCallback callba
             callback.callback(false, null, -1, null);
         } else {
             params.doLogin();
+            MainActivity.loadingBar.setVisibility(View.VISIBLE);
```

Comment:
Natürlich kann man das auch in den jeweiligen Callbacks handeln. Dann muss aber JEDES Fragment / Modul in jeder Callback Methode das Teil ausblenden, wenn es das vorher eingeblendet hat und außerdem immer gucken, ob noch andere Module irgendwas laden.

Ich könnte mir vorstellen, das in eine statische Methode der MyTFG Klasse auszulagern?! Die kann dann zur Not entscheiden, ob grade ein UI da ist oder nicht.

Eine Ladeanimation für die Fragmente an sich können wir auch noch machen, einen drehenden Kringel oder so. Dann ist oben der Balken nur für die API (wäre ein Vorschlag)